Routes/UserRoute.js

```diff
@@ -53,4 +53,19 @@ router.route("/adduser").post((req, res) => {
     });
 });
 
+//Endpoint to delete a new user
+router.route("/delete/:id").delete((req, res) => {
+  user
+    .findByIdAndDelete(req.params.id)
+    .then((user) => {
+      if (!user) {
+        return res.status(404).json("User not Find with ID:" + user);
+      }
+      res.send(user);
```

Comment:
Send string saying user is Deleted successfully with his id instead of the user object of the deleted user